### PR TITLE
docs: add v2.9.0 changelog

### PR DESCRIPTION
**Pull request by bpmct**

### CODE REVIEW

File: docs/changelogs/v2.9.0.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+#### Backend
+
+- Expose DERP server debug metrics (#12135) (@spikecurtis)
+- Add logSender for sending logs on agent v2 API (#12046) (@spikecurtis)
+- Clean up organization handling (#12142) (#12143) (#12146) (@Emyrk)
+- Change agent to use v2 API for reporting stats (#12024) (@spikecurtis)
+- Send log limit exceeded in response, not error (#12078) (@spikecurtis)
+- Add template activity_bump property (#11734) (@deansheather)
+- Add activity status and autostop reason to workspace overview (#11987) (@aslilac)
```
Comment (stirby) [inline docs/changelogs/v2.9.0.md:55]: Shouldn't this be in the dashboard section?

**Review by stirby — APPROVED** [review]
Looks good! Massive release.

File: docs/changelogs/v2.9.0.md (hunk 1 above)
Comment (stirby) [reply docs/changelogs/v2.9.0.md]: Moved this to the dashboard and added a screenshot.

Comment (bpmct) [reply docs/changelogs/v2.9.0.md]: yep!

**Review by stirby — APPROVED** [review]
👍

**Pull request merged**